containers/unit_tests/TestOffsetView.hpp

```diff
@@ -278,6 +278,79 @@ void test_offsetview_construction(unsigned int size) {
 #endif
   }
 }
+
+template <typename Scalar, typename Device>
+void test_offsetview_unmanaged_construction() {
+  // Preallocated memory (Only need a valid address for this test)
+  Scalar s;
+
+  {
+    // Constructing an OffsetView directly around our preallocated memory
+    Kokkos::Array<int64_t, 1> begins1{2};
+    Kokkos::Array<int64_t, 1> ends1{3};
```

Comment:
Also Travis CI builds are [failing on OSX](https://travis-ci.org/kokkos/kokkos/builds/584868385?utm_source=github_status&utm_medium=notification).  Apple LLVM version 9.1.0 is raising a few warnings:
```
error: suggest braces around initialization of subobject [-Werror,-Wmissing-braces]
```
I guess you can just use double brace initialization to fix it.

Reply:
Will do.